docs/markdown/Go/go.md

```diff
@@ -186,21 +188,23 @@ go_package(dependencies=[":testdata"])
 
 files(name="testdata", sources=["testdata/*"])
 ```
+
 ```go pkg/runner/foo_test.go
 package foo
 
 import (
-	"os"
-	"testing"
+ "os"
+ "testing"
 )
 
 func TestFilesAvailable(t *testing.T) {
-	_, err := os.Stat("testdata/f.txt")
-	if err != nil {
-		t.Fatalf("Could not stat pkg/runner/testdata/f.txt: %v", err)
-	}
+ _, err := os.Stat("testdata/f.txt")
+ if err != nil {
+  t.Fatalf("Could not stat pkg/runner/testdata/f.txt: %v", err)
+ }
```

Comment:
Bad change, given that Go is tab based?

Reply:
Yeah, I use VS Code, and it doesn't handle mixed indentation very well. I expect it "helpfully" formatted the doc for me.